src/Ng/SearchIndexFromCatalogCollector.cs

```diff
@@ -53,7 +53,7 @@ public SearchIndexFromCatalogCollector(Uri index, Lucene.Net.Store.Directory dir
 
                 indexWriter.ExpungeDeletes();
 
-                indexWriter.Commit(CreateCommitMetadata(commitTimeStamp));
+                indexWriter.Commit(DocumentCreator.CreateCommitMetadata(commitTimeStamp, "from catalog", Guid.NewGuid().ToString()));
```

Comment:
Please make sure you have test to confirm the component who reading this metadata compatible with this change.